static/js/factories/search.js

```diff
@@ -66,6 +68,27 @@ export const makePostResult = (): PostResult => ({
   text:                casual.text
 })
 
+export const makeCourseResult = (): CourseResult => ({
+  course_id:         `course_${String(casual.random)}`,
+  title:             casual.title,
+  image_src:         "http://image.medium.url",
+  short_description: casual.description,
+  full_description:  casual.description,
+  platform:          casual.random_element([platforms.edX, platforms.OCW]),
+  language:          casual.random_element(["en-US", "fr", null]),
+  semester:          casual.random_element(["Fall", "Spring", null]),
+  year:              casual.year,
+  level:             casual.random_element(["Graduate", "Undergraduate", null]),
+  start_date:        casual.date("YYYY-MM-DD[T]HH:mm:ss[Z]"),
+  end_date:          casual.date("YYYY-MM-DD[T]HH:mm:ss[Z]"),
+  enrollment_start:  casual.date("YYYY-MM-DD[T]HH:mm:ss[Z]"),
+  enrollment_end:    casual.date("YYYY-MM-DD[T]HH:mm:ss[Z]"),
+  instructors:       ["instuctor 1", "instructor 2"],
+  topics:            [casual.word, casual.word],
+  prices:            [{ mode: "audit", price: casual.number }],
+  object_type:       "course"
+})
+
 export const makeSearchResult = (type: ?string) => {
   if (!type) {
     type = casual.random_element(["post", "comment", "profile"])
```

Comment:
Can you add "course" here?

Reply:
I think it would be best not to, since this might bring back course results on the default search page when no particular type is selected.

Reply:
Actually I think this is just for tests here